### DISCUSSION TITLE
pulling metars & tafs using python

### DISCUSSION THREAD

**Discussion by cardboardsinxoverx** [weather-gov/api#760] [Q&A]
hi yall, i've been creating a bot for my personal discord for all things weather, and I have quite a few goodies in there. This wasn't always a problem though, it wasn't until ADDS updated their website and API stuff that this happened. suddenly, I'm not able to pull more than one metar from a station (say i want six hours of obs, that's what I ask for, and I will only get one observation). no errors show up in the logs.

then tafs, uh this doesn't work at all. I've done ALL i can do with error handling. when I do a command like $taf kvpc, nothing will happen.

anyone else having this issue with ADDS API? im really new at this so im unsure how to recreate my issue by looking around for it in other threads.

**Comment by dwhitemv25** [comment]
Sorry, but what is ADDS? Google doesn't seem to know.

**Reply by akrherz** [reply]
I believe ADDS was this: https://ral.ucar.edu/solutions/products/experimental-aviation-digital-data-service-adds

**Reply by dwhitemv25** [reply]
The linked project was shut down in 2015 and the Operational link there goes to aviationweather.gov. So I don't know if OP's question is in relation to that API or not. As per yesterday's thread #759, TAFs from api.weather.gov are not returning data but TAFs from aviationweather.gov are working fine, so maybe that is part of the problem?
